rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/tsdb/25_id_generation.yml

```diff
@@ -152,6 +152,55 @@ generates a consistent id:
   - match: { hits.hits.8._source.@timestamp: 2021-04-28T18:52:04.467Z }
   - match: { hits.hits.8._source.k8s.pod.uid: 947e4ced-1786-4e53-9e0c-5c447e959507 }
 
+  - do:
```

Comment:
This repeats the previous search in the test (to keep the test setup) but
adds the necessary conditions (sorting and search_after) in order
to exercise the modified code.

Reply:
Oops, fix incoming to add the necessary version skips